### ISSUE TITLE
Other authentication method

### ISSUE THREAD

**Issue by sakama** [embulk/embulk-output-bigquery#2]
[Predefined access token](https://github.com/kaizenplatform/fluent-plugin-bigquery#authentication) (Compute Engine only) support?
## private_key #default

```
out:
  type: bigquery
  auth_method private_key
  service_account_email xxxxx@developer.gserviceaccount.com
  p12_keyfile_path /path/to/privatekey.p12
```
## predefined access token

```
out:
  type: bigquery
  auth_method: compute_engine
```

**Comment by sakama** [comment]
authorization of predefined access token was implemented at v0.1.3

**Comment by sakama** [comment]
auth method using json_key of GCE service account was also supported at v0.1.10
